### ISSUE TITLE
blinds - Unable to validate value

### ISSUE THREAD

**Issue by andrasg** [arachnetech/homebridge-mqttthing#428]
Hello,

I have set up a blind with the following config:

```json
{
    "accessory": "mqttthing",
    "type": "windowCovering",
    "name": "terraceBlinds",
    "url": "mqtt://192.168.20.15:1883",
    "topics": {
        "getCurrentPosition": "homebridge/status/blindposition/terrace",
        "setTargetPosition": "homebridge/control/blindtargetposition/terrace",
        "getTargetPosition": "homebridge/status/blindtargetposition/terrace"
    }
}
```

When closing the blinds, I am feeding an int (`0`) to the target position and then every second a status update in int to the current position topic. The current position seem to not like the value though: 

```[4/11/2021, 5:02:49 PM] [terraceBlinds] Unable to validate Current Position, format [uint8] - {"_events":{},"_eventsCount":3,"iid":10,"value":93,"status":null,"statusCode":0,"subscriptions":3,"displayName":"Current Position","UUID":"0000006D-0000-1000-8000-0026BB765291","props":{"format":"uint8","perms":["ev","pr"],"unit":"percentage","minValue":0,"maxValue":100,"minStep":1}}```

When feeding the int converted to a string first, the error message doesn't seem to be displayed.

Why am I getting this error?

**Comment by andrasg** [comment]
Could be a duplicate of #426

**Comment by arachnetech** [comment]
Should be fixed in 1.1.28.
